src/lib.rs

```diff
@@ -1,5 +1,6 @@
-#![deny(missing_docs)]
+//#![deny(missing_docs)]
```

Comment:
any reason this is commented out?

Reply:
Just commented it out because I was removing/adding different structures often, I'll uncomment and document stuff once I figure out the final design for it.